tests/timeseries/test_logical_meter.py

```diff
@@ -328,24 +343,31 @@ async def test_formula_composition(  # pylint: disable=too-many-locals
         grid_power_recv = await logical_meter.grid_power()
         battery_power_recv = await logical_meter.battery_power()
         pv_power_recv = await logical_meter.pv_power()
+        ev_power_recv = await logical_meter.ev_charger_power()
 
-        engine = (pv_power_recv.clone() + battery_power_recv.clone()).build("inv_power")
+        engine = (
+            pv_power_recv.clone() + battery_power_recv.clone() + ev_power_recv.clone()
+        ).build("inv_power")
         inv_calc_recv = engine.new_receiver()
```

Comment:
@sahas-subramanian-frequenz is it correct to include `ev_power` as part of `inv_power`?

Reply:
We agreed on create a dedicated test for EV power